save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -36,11 +37,13 @@ class FixPlugin(testConfig: TestConfig, testFiles: List<String> = emptyList()) :
 
     override fun handleFiles(files: Sequence<List<Path>>): Sequence<TestResult> {
         val fixPluginConfig = testConfig.pluginConfigs.filterIsInstance<FixPluginConfig>().single()
+        val generalConfig = testConfig.pluginConfigs.filterIsInstance<GeneralConfig>().singleOrNull()
         logInfo("Discovered the following file pairs for comparison: $files")
         return files
             .map { it.first() to it.last() }
             .map { (expected, test) ->
-                val executionResult = pb.exec(fixPluginConfig.execCmd, null, false)
+                val execCmd = generalConfig?.execCmd + " " + fixPluginConfig.execFlags
```

Comment:
What should we do, if general config is null? I think, we should either raise a warning or use `execFlags` only instead of `execCmd`